src/config/parse.auto.c

```diff
@@ -2040,6 +2346,33 @@ static const struct option optionList[] =
         .has_arg = required_argument,
         .val = PARSE_OPTION_FLAG | PARSE_DEPRECATE_FLAG | (0 << PARSE_KEY_IDX_SHIFT) | cfgOptRepoCipherType,
     },
+    {
+        .name = "repo2-cipher-type",
+        .has_arg = required_argument,
+        .val = PARSE_OPTION_FLAG | (1 << PARSE_KEY_IDX_SHIFT) | cfgOptRepoCipherType,
+    },
+    {
+        .name = "reset-repo2-cipher-type",
+        .val = PARSE_OPTION_FLAG | PARSE_RESET_FLAG | (1 << PARSE_KEY_IDX_SHIFT) | cfgOptRepoCipherType,
+    },
+    {
+        .name = "repo3-cipher-type",
+        .has_arg = required_argument,
+        .val = PARSE_OPTION_FLAG | (2 << PARSE_KEY_IDX_SHIFT) | cfgOptRepoCipherType,
+    },
+    {
+        .name = "reset-repo3-cipher-type",
+        .val = PARSE_OPTION_FLAG | PARSE_RESET_FLAG | (2 << PARSE_KEY_IDX_SHIFT) | cfgOptRepoCipherType,
+    },
+    {
+        .name = "repo4-cipher-type",
+        .has_arg = required_argument,
+        .val = PARSE_OPTION_FLAG | (3 << PARSE_KEY_IDX_SHIFT) | cfgOptRepoCipherType,
+    },
+    {
+        .name = "reset-repo4-cipher-type",
+        .val = PARSE_OPTION_FLAG | PARSE_RESET_FLAG | (3 << PARSE_KEY_IDX_SHIFT) | cfgOptRepoCipherType,
+    },
 
     // repo-default option
     // -----------------------------------------------------------------------------------------------------------------------------
```

Comment:
What happens when I use reset-repo-default? This question may be out of scope for the current version of the feature, but I'm trying to understand what would happen...

Reply:
See below.